source/modules/MEI.xml

```diff
@@ -3022,8 +3064,20 @@
             </valItem>
           -->
         <valItem ident="tab">
-          <desc xml:lang="en">Tablature notation.</desc>
+          <desc xml:lang="en">Generic tablature notation.</desc>
+        </valItem>
+        <valItem ident="tab.guitar">
+          <desc xml:lang="en">Tablature notation for guitars (includes "spanish" lute tablature). Numbers for frets and courses closest to the ground when playing are at the top of the stave.</desc>
```

Comment:
Perhaps a definition of "ground" is needed somewhere?

Reply:
Intended as "closest to the floor" (i.e., maximal physical proximity to the centre of the earth!). The notion of "lowest" is annoyingly difficult to convey without any ambiguity in this context

Reply:
A diagram or two somewhere can be helpful. "Bottom" vs "Top" may also be useful, along with maybe ensuring "lowest pitch" is always qualified as such.

Reply:
We have gone for the pragmatic description, "closest to the player's feet". I hope this is now clear :-)
We have also clarified other wording a little.